api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -554,32 +525,18 @@ public List<ReportingWorkspace> getWorkspaceBatch(long limit, long offset) {
             + "  LEFT JOIN access_tier a ON c.access_tier = a.access_tier_id\n"
             // most workspaces are not Featured
             + "  LEFT OUTER JOIN featured_workspace fw ON w.workspace_id = fw.workspace_id\n"
-            // not all users have initial credits expiration entries
-            + "  LEFT OUTER JOIN user_initial_credits_expiration uice ON uice.user_id = creator_id\n"
-            // some users don't have institutional affiliations
-            + "  LEFT JOIN user_verified_institutional_affiliation uvia ON w.creator_id = uvia.user_id\n"
-            + "  LEFT JOIN institution i ON uvia.institution_id = i.institution_id\n"
             + "WHERE active_status = ? \n"
             + "ORDER BY w.workspace_id\n"
             + "LIMIT ? \n"
             + "OFFSET ?";
     return jdbcTemplate.query(
         sql,
-        new Object[] {
```

Comment:
this call signature `query(sql, Object[] params, mapper)` is deprecated in favor of `query(sql, mapper, Object... params)`.  I removed the first 3 no-longer-used params, and shifted the last 3 to the end of the signature.